src/Json_encode.mli

```diff
@@ -45,18 +45,21 @@ val tuple4 : 'a encoder -> 'b encoder -> 'c encoder -> 'd encoder -> ('a * 'b *
 external dict : Js.Json.t Js_dict.t -> Js.Json.t = "%identity"
 (** [dict d] makes a JSON object of the [Js.Dict.t] [d] *)
 
+val dictOf : 'a encoder -> 'a Js_dict.t encoder
+(** [dict(enc) d] makes a JSON object of the [Js.Dict.t] [d] with the given encoder *)
```

Comment:
This should be "[dictOf encoder d]", and "...with the given [encoder]" to be consistent with the others.

Reply:
Thanks! Yeah I kinda half-assed that one :)